extensions/knative/client/src/main/java/io/fabric8/knative/client/DefaultKnativeClient.java

```diff
@@ -157,127 +157,127 @@ public NamespacedKnativeClient inNamespace(String namespace) {
     }
 
     @Override
-    public MixedOperation<Service, ServiceList, DoneableService, Resource<Service, DoneableService>> services() {
+    public MixedOperation<Service, ServiceList, Resource<Service>> services() {
         return new ServiceOperationsImpl(this.getHttpClient(), this.getConfiguration());
     }
 
     @Override
-    public MixedOperation<Route, RouteList, DoneableRoute, Resource<Route, DoneableRoute>> routes() {
+    public MixedOperation<Route, RouteList, Resource<RouteRoute>> routes() {
```

Comment:
That, and subsequent doesn't look right… `RouteRoute`?

Reply:
No it doesn't